RedditSharp/Reddit.cs

```diff
@@ -265,7 +265,7 @@ public AuthenticatedUser GetMe()
         /// <returns></returns>
         public Subreddit GetSubreddit(string name)
         {
-            name = System.Text.RegularExpressions.Regex.Replace(name, "(r/|/)", "");
+            name = System.Text.RegularExpressions.Regex.Replace(name, "^(r/|/)", "");
```

Comment:
Close, but still not quite right. Try it out on http://www.regexpal.com

"/r/subredditr/" => "r/subredditr/"

I think this would be the correct regex `"(^[/]?[r]?[/])|[/]"`

that would replace any leading "r/" or "/r/" and then any other slashes existing.